### DISCUSSION TITLE
Disable LHRTUNE on Newer 3080 12gb cards

### DISCUSSION THREAD

**Discussion by DeezMine154** [Lolliedieb/lolMiner-releases#1317] [General]
I'm trying to dual mine eth+ton on a 2 card rig. One 3080 12gb latest version, and the other is 2070s.  With T-rex you can trick the 3080 to not get seen as LHR card and you can maintain at least a 63mh/s rate. Is there a way to also do this with LolMiner? My card is always getting locked ,paused resume locked while decreasing hashrate each time.

**Comment by michiganpinball** [comment]
I had same issue.  Use the --lhrtune off parameter to disable the LHR checking.
